Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -543,6 +550,9 @@ final class PurchasesOrchestrator {
         }
 
         payment.applicationUsername = self.appUserID
+        if let quantity = quantity {
+            payment.quantity = quantity
```

Comment:
Interesting... so in Apple devs can specify the number, whereas in Android, the customer needs to specify the quantity 😫